lib/ffaker_wrapper.rb

```diff
@@ -1,42 +1,90 @@
 require "ffaker"
 
 module FfakerWrapper
-  def date_string
-    date.strftime("%F")
-  end
+  STATUS = {
+    0 => "Open",
+    1 => "Closed",
+    2 => "Recommended for closure",
+    3 => "Recommended to be reopened",
+    4 => "Reopened"
+  }
+
+  ISSUE_TYPE = {
+    1 => "Complaint",
+    8 => "COVID-19 Concern"
+  }
+
+  PRIORITY = {
+    0 => "Urgent",
+    1 => "Routine",
+    2 => "FYI"
+  }
 
   def date
     FFaker::Time.between(1.month.ago, 1.month.after).to_date
   end
 
-  def datetime_string
-    # format UTC date YYYY-MM-DDTHH:MM:SSZ
-    datetime.strftime("%FT%TZ")
+  def date_string
+    date.strftime("%F")
   end
 
   def datetime
     FFaker::Time.between(1.month.ago, 1.month.after).to_datetime
   end
 
+  def datetime_string
+    # format UTC date YYYY-MM-DDTHH:MM:SSZ
+    datetime.strftime("%FT%TZ")
+  end
+
   def grantee_name
     # these just sound cool for use as the grantee name :)
     FFaker::AddressUK.street_name
   end
 
+  def grant_number
+    # Note: actual grant numbers appear like 04CH011504
+    random_int(99999).to_s
+  end
+
   # returns a string because actual identifiers may not be integers
   def identifier
     random_int(9999).to_s
   end
 
+  def issue_type_object
+    id = ISSUE_TYPE.keys.sample
+    {
+      id: id,
+      label: ISSUE_TYPE[id]
+    }
+  end
+
   def phrase
     FFaker::Lorem.phrase
   end
 
+  def priority_object
+    id = random_int(2)
+    {
+      id: id,
+      label: PRIORITY[id]
+    }
+  end
+
   def random_int(range = 3)
     FFaker::Random.rand(range)
   end
 
-  def word
-    FFaker::Lorem.word
+  def regarding
+    ["Grantee", "State", "Region", "DRS"].sample
```

Comment:
any reason not to make this a constant like `ISSUE_TYPE`?

Reply:
I think I didn't it since there aren't identifiers involved like there were with the others, but there's no good reason not to make this a constant, either!